sync.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+MONGO_PERSIST="-mongo"
+NIGHT_BUILD_URL="https://raw.githubusercontent.com/lenny-intel/developer-scripts/multi2/releases/nightly-build/compose-files"
```

Comment:
Would this change to `edgexfoundry` later?

Reply:
Yes, once that PR has been merged, I will update the link and merge this PR.